packages/plotly-demo/src/heatmap.js

```diff
@@ -0,0 +1,92 @@
+import Plotly from 'plotly.js-dist';
+import {
+  generateBasicAnnotations,
+  ahoi,
+  EVisualizationType,
+} from '@visahoi/plotly';
+import debounce from 'lodash.debounce';
+
+let chart = null;
+let showOnboarding = false;
+let onboardingUI = null;
+
+const debouncedResize = debounce((event) => {
+  onboardingUI?.updateOnboarding(getAhoiConfig());
+}, 250);
+
+const render = async () => {
+  chart = await makePlotly();
+  window.addEventListener('resize', debouncedResize);
+};
+
+const makePlotly = () => {
+  const data = [
+    {
+      z: [
+        [1, null, 30, 50, 1],
```

Comment:
Please use the following data: 
```[
        [14, null, 19, 24, 16],
        [17, 15, 28, 33, 20],
        [19, 23, 29, 18, 18],
      ] ```

      
It makes more sense in terms of °C.